cli/package.json

```diff
@@ -60,6 +60,7 @@
     "semver": "^7.5.3",
     "supports-color": "^8.1.1",
     "tmp": "~0.2.3",
+    "tree-kill": "1.2.2",
```

Comment:
@jennifer-shehane should we mention in the changelog that we are adding the `tree-kill` dependency to the CLI since we mention dependency updates?

Reply:
I can't recall adding a line for added dependencies before